cmd/skywire-visor/commands/root.go

```diff
@@ -380,7 +378,7 @@ func initConfig(mLog *logging.MasterLogger, confPath string) *visorconfig.V1 { /
 		log.WithError(err).Fatal("Failed to read in config.")
 	}
 	if !compat {
-		updateConfig(mLog, conf)
+		updateConfig(mLog)
```

Comment:
This can be one line of just using log.Fatalf() for instance. Seems overkill to me to assign a new package logger and then log error and fatal separately.

Reply:
you are right, separate func was vestigial - fixed in [cd88cfe](https://github.com/skycoin/skywire/pull/1157/commits/cd88cfecedfd287b1ca7ae221b4b02b7a359ad0c)